packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -58,8 +59,13 @@ class MethodChannelCamera extends CameraPlatform {
   @override
   Future<List<CameraDescription>> availableCameras() async {
     try {
-      final List<Map<dynamic, dynamic>> cameras = await _channel
+      final cameras = await _channel
```

Comment:
I think we want left-hand types (but I'm not 100% sure), @bparrishMines?

Reply:
It differs between plugins. If the general style within the plugin has left-hand types, then we should probably keep the pattern the same. We could also decide on a preference for the repo as a whole and add it to the global `analysis_options.yaml`. https://github.com/flutter/plugins/blob/master/analysis_options.yaml